src/clr-addons/slider/slidersize.enum.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export enum SliderSize {
```

Comment:
if your class/enum in camelcase is "SliderSize" then your file name would be slider-size.enum.ts (fix for all)